policies/templates/gcp_network_restrict_default_v1.yaml

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: templates.gatekeeper.sh/v1alpha1
+kind: ConstraintTemplate
+metadata:
+  name: gcp-network-restrict-default-v1
+spec:
+  crd:
+    spec:
+      names:
+        kind: GCPNetworkRestrictDefaultV1
+      validation:
+        openAPIV3Schema:
+          mode:
+            type: string
+            default: GLOBAL
+  targets:
+    validation.gcp.forsetisecurity.org:
+      rego: | #INLINE("validator/network_restrict_default.rego")
+            # Copyright 2019 Google LLC
+            #
+            # Licensed under the Apache License, Version 2.0 (the "License");
+            # you may not use this file except in compliance with the License.
+            # You may obtain a copy of the License at
+            #
+            #			http://www.apache.org/licenses/LICENSE-2.0
+            #
+            # Unless required by applicable law or agreed to in writing, software
+            # distributed under the License is distributed on an "AS IS" BASIS,
+            # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+            # See the License for the specific language governing permissions and
+            # limitations under the License.
+            #
+            
+            package templates.gcp.GCPNetworkRestrictDefaultV1
+            
+            import data.validator.gcp.lib as lib
+            
+            deny[{
+            	"msg": message,
+            	"details": metadata,
+            }] {
+            	a := input.asset
+            	count({a.asset_type} & {"compute.googleapis.com/Network", "google.compute.Network"}) == 1
```

Comment:
Why this test? Is it for backward compatibility?

Reply:
Probably copypasta error.